fe1-web/src/features/evoting/screens/__tests__/CreateElection.test.tsx

```diff
@@ -45,10 +72,96 @@ describe('CreateElection', () => {
 
   it('renders correctly when name and question are empty', () => {
     const component = render(
-      <FeatureContext.Provider value={contextValue}>
-        <MockNavigator component={CreateElection} />
-      </FeatureContext.Provider>,
+      <Provider store={mockStore}>
+        <FeatureContext.Provider value={contextValue}>
+          <MockNavigator component={CreateElection} />
+        </FeatureContext.Provider>
+      </Provider>,
     ).toJSON();
     expect(component).toMatchSnapshot();
   });
+
+  it('renders correctly when name and questions are whitespace string', () => {
+    const { getByTestId, toJSON } = render(
+      <Provider store={mockStore}>
+        <FeatureContext.Provider value={contextValue}>
+          <MockNavigator component={CreateElection} />
+        </FeatureContext.Provider>
+      </Provider>,
+    );
+    const electionName = getByTestId('election_name_selector');
+    const question = getByTestId('question_selector_0');
+    fireEvent.changeText(electionName, '     ');
+    fireEvent.changeText(question, '        ');
+    expect(toJSON()).toMatchSnapshot();
```

Comment:
I think waiting here is necessary for the snapshot to properly reflect the changes? 😅

Reply:
It is not necessary.